agent/tool-scripts/jmap.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
```

Comment:
This script has to be named jmap instead of jmap.sh and needs to have executable permission.